app/models/concerns/db_activity_methods.rb

```diff
@@ -0,0 +1,38 @@
+module DbActivityMethods
+  extend ActiveSupport::Concern
+
+  included do
+    def save_and_create_db_activity(user, action)
```

Comment:
Assignment Branch Condition size for save_and_create_db_activity is too high. [17.29/15]